alarm/binance_utils.py

```diff
@@ -6,16 +6,33 @@
 
 logger = logging.getLogger(f'{__name__}')
 
+binance_exchange_info_url = "https://api.binance.com/api/v3/exchangeInfo"
 
-def get_binance_trade_pairs():
-    binance_exchange_info_url = "https://api.binance.com/api/v3/exchangeInfo"
+
+def format_trade_pair_for_user_message(trade_pair):
```

Comment:
Should be done for the whole phone number.

Reply:
++